diff --git a/inflect/__init__.py b/inflect/__init__.py
index 3eef486..2d66717 100644
--- a/inflect/__init__.py
+++ b/inflect/__init__.py
@@ -1986,7 +1986,7 @@ DENOMINATOR = re.compile(r"(?P<denominator>.+)( (per|a) .+)")
 PLVERB_SPECIAL_S_RE = re.compile(rf"^({plverb_special_s})$")
 WHITESPACE = re.compile(r"\s")
 ENDS_WITH_S = re.compile(r"^(.*[^s])s$", re.IGNORECASE)
-ENDS_WITH_APOSTROPHE_S = re.compile(r"^(.+)'s?$")
+ENDS_WITH_APOSTROPHE_S = re.compile(r"^(.*)'s?$")
 INDEFINITE_ARTICLE_TEST = re.compile(r"\A(\s*)(?:an?\s+)?(.+?)(\s*)\Z", re.IGNORECASE)
 SPECIAL_AN = re.compile(r"^[aefhilmnorsx]$", re.IGNORECASE)
 SPECIAL_A = re.compile(r"^[bcdgjkpqtuvwyz]$", re.IGNORECASE)
