diff --git a/inflect/__init__.py b/inflect/__init__.py
index 3eef486..7192314 100644
--- a/inflect/__init__.py
+++ b/inflect/__init__.py
@@ -1822,7 +1822,7 @@ plverb_ambiguous_non_pres = re.compile(
 
 pl_v_oes_oe = ("canoes", "floes", "oboes", "roes", "throes", "woes")
 pl_v_oes_oe_endings_size4 = ("hoes", "toes")
-pl_v_oes_oe_endings_size5 = ("shoes",)
+pl_v_oes_oe_endings_size5 = "shoes"
 
 
 pl_count_zero = ("0", "no", "zero", "nil")
